workloads/templates/common-templates.md

```diff
@@ -134,43 +135,87 @@ in templates do not automatically affect VMs, or vice versa).
 
 ## common template customization
 
-### template customization - memory, CPU
+The templates provided by the kubevirt project provide a set of conventions and annotations that augment the basic feature of the
+[openshift templates](https://docs.okd.io/latest/dev_guide/templates.html).
+You can customize your kubevirt-provided templates editing these annotations, or you can add them to your existing templates to
+make them consumable by the kubevirt services.
 
-There are three options to customize VM memory and CPU:
+Here's a description of the kubevirt annotations. Unless otherwise specified, the following keys are meant to be top-level entries
+of the template metadata, like
 
- * select flavor - tiny, small, medium, etc. Each flavor grants to VM different amount of RAM and CPU cores.
- * setting up directly editable fields spec.template.spec.domain.cpu.cores and spec.template.spec.domain.resources.requests.memory, example:
+```yaml
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-10
+  annotations:
+    openshift.io/display-name: "Generic demo template"
+```
 
-```bash
-oc patch virtualmachine testguest --type merge -p '{"spec":{"template":{"spec":{"domain":{"cpu":{"cores":'3'}}}}}}'
-virtualmachine.kubevirt.io/testguest patched
+All the following annotations are prefixed with `defaults.template.cnv.io`, which is omitted below for brevity. So the actual annotations you should use will look like
+
+```yaml
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-10
+  annotations:
+    defaults.template.cnv.io/disk: default-disk
+    defaults.template.cnv.io/volume: default-volume
+    defaults.template.cnv.io/nic: default-nic
+    defaults.template.cnv.io/network: default-network
 ```
- * use WebUI: WebUI supports different flavors and Workload Profiles. Also it supports custom amount of memory and CPUs
 
-### template customization - networking
+Unless otherwise specified, all annotations are meant to be safe defaults, both for performance and compability, and hints for the CNV-aware UI and tooling.
 
-There are two options to customize Networking in VM:
- * setting up directly editable field(see example above): spec.template.spec.networks. You can edit following:
-   * name (of interface)
-   * pod
- * edit networking in webUI. Please note: WebUI also edits spec.domain.interfaces and supports changing mac address
+#### disk
+default disk type to use. It is recommended to aim for maximum compatibility.
```

Comment:
The value is not the disk type, but a name of disk that serves as a template.

Reply:
Interesting, I misunderstood https://github.com/kubevirt/common-templates/blob/master/templates/README.md, in particular """    # The goal of default disk is to define what kind of disk
    # is supported by the OS mainly in terms of bus (ide, scsi,
    # sata, virtio, ...)
    defaults.template.cnv.io/disk: default-disk"""

and, as you probably guessed, this error was replicated for all the other cases below. Solving them all.